torch/_inductor/codecache.py

```diff
@@ -553,8 +560,14 @@ class FxGraphHashDetails:
     # Excluded kwargs param that are not stable between runs
     EXCLUDED_KWARGS = ["graph_id"]
 
-    def __init__(self, fx_args: List[Any], fx_kwargs: Dict[str, Any]) -> None:
-        self.fx_args = fx_args
+    def __init__(
+        self,
+        gm: torch.fx.GraphModule,
```

Comment:
Since we need to access the `example_args`, it was convenient to make the `gm` and `example_args` arguments explicit rather than packaging them up in a list.